bangazonapi/views/product.py

```diff
@@ -102,6 +103,12 @@ def create(self, request):
 
             new_product.image_path = data
 
+        try:
+            new_product.clean_fields(exclude='image_path')
+        except ValidationError as ex:
+                return Response({"msg": ex}, status=status.HTTP_400_BAD_REQUEST)
+
+
         new_product.save()
```

Comment:
Would recommend putting this save inside the `try` block. If you want to add another `except` block in case the `save` fails, then that would also be acceptable.

Reply:
done and done!